rooms.js

```diff
@@ -163,6 +165,107 @@ var Room = (function () {
 		}
 		return true;
 	};
+	//mute handling
+	Room.prototype.runMuteTimer = function () {
+		if (this.muteTimer || this.mutedUsers.length === 0) return;
```

Comment:
Is there a way to link to lines so it changes dynamically with updates? That seems like it would be useful.